src/backend-plugin.ts

```diff
@@ -44,6 +54,16 @@ export class WebpackBackend implements Module {
                 return Promise.all(
                     languages.map(async lang => {
                         const builtKey = `./${lang}/${namespace}.json`;
+                        if (this.jsons == null) {
+                            callback(
+                                new Error(`No context given!`),
+                                // TODO: Fix this when types are up to date with newest implementation.
+                                // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+                                // @ts-ignore
+                                null
+                            );
+                            return;
+                        }
```

Comment:
Move this check to assert function.
You will no longer need to duplicate check.
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions